<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QtInvestigateMyClickWinnersMainDialog</class>
 <widget class="QDialog" name="QtInvestigateMyClickWinnersMainDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>639</width>
    <height>603</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>InvestigateMyClickWinners</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QWidget" name="widget" native="true">
     <layout class="QGridLayout" name="gridLayout">
      <item row="3" column="0" colspan="2">
       <widget class="QLabel" name="label_focal_customer">
        <property name="text">
         <string>Focal customer</string>
        </property>
       </widget>
      </item>
      <item row="7" column="0">
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="0" column="0" colspan="2">
       <widget class="QLabel" name="label_company">
        <property name="text">
         <string>Company</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QDoubleSpinBox" name="box_profit_webshop_euro_per_year">
        <property name="maximum">
         <double>1000000.000000000000000</double>
        </property>
        <property name="value">
         <double>100.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="8" column="0" colspan="2">
       <widget class="QPushButton" name="button_run">
        <property name="text">
         <string>Run</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_profit_website">
        <property name="text">
         <string>Profit website (euro per month)</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QDoubleSpinBox" name="box_profit_website_euro_per_month">
        <property name="decimals">
         <number>2</number>
        </property>
        <property name="maximum">
         <double>1000000.000000000000000</double>
        </property>
        <property name="value">
         <double>100.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="label_n_membership_years">
        <property name="text">
         <string>#membership years</string>
        </property>
       </widget>
      </item>
      <item row="5" column="0" colspan="2">
       <widget class="QLabel" name="label_other_customer">
        <property name="text">
         <string>Other customers</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_profit_webshop">
        <property name="text">
         <string>Profit webshop (euro per year)</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QSpinBox" name="box_n_membership_years">
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="value">
         <number>1</number>
        </property>
       </widget>
      </item>
      <item row="6" column="1">
       <widget class="QSpinBox" name="box_n_other_customers">
        <property name="maximum">
         <number>1000000</number>
        </property>
        <property name="value">
         <number>1</number>
        </property>
       </widget>
      </item>
      <item row="6" column="0">
       <widget class="QLabel" name="label_n_other_customers">
        <property name="text">
         <string>#other customers</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="widget_plots" native="true">
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QwtPlot" name="plot_focal_person" native="true"/>
      </item>
      <item>
       <widget class="QwtPlot" name="plot_company" native="true"/>
      </item>
      <item>
       <widget class="QPlainTextEdit" name="text_transactions">
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>400</height>
         </size>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QwtPlot</class>
   <extends>QWidget</extends>
   <header>qwt_plot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>button_run</tabstop>
  <tabstop>box_profit_website_euro_per_month</tabstop>
  <tabstop>box_profit_webshop_euro_per_year</tabstop>
  <tabstop>box_n_membership_years</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
